modules/core/include/opencv2/core/core_c.h

```diff
@@ -1490,6 +1490,7 @@ CVAPI(void)  cvNormalize( const CvArr* src, CvArr* dst,
 #define CV_REDUCE_AVG 1
 #define CV_REDUCE_MAX 2
 #define CV_REDUCE_MIN 3
+#define CV_REDUCE_SUM2 4
```

Comment:
Please don't touch ".c" files anymore (deprecated).

Reply:
Ok, but in that case I will have to change all references to CV_REDUCE_* in cpp code, in favor of cv::REDUCE_*

Reply:
Ok, please use enum values: `REDUCE_` (`cv::` prefix is not needed)